packages/integration-sdk-cli/src/commands/collect.ts

```diff
@@ -41,6 +41,10 @@ export function collect() {
       'Can be used with the `--step` to specify a path to a non-default cache location.',
     )
     .option('-V, --disable-schema-validation', 'disable schema validation')
+    .option(
+      '-b, --buffer-size <number>',
+      'specify buffer size for use in data collection (default 500)',
```

Comment:
Could we add units? I assume this means 500 objects?

Reply:
Updated in latest commit.